utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -92,3 +92,70 @@ const val CACHE_LATEX_RENDERING = "cache_latex_rendering"
 
 /** Default value for whether to cache LaTeX rendering using Glide. */
 const val CACHE_LATEX_RENDERING_DEFAULT_VALUE = true
+
+/** Qualifier for the platform parameter that controls whether to record performance metrics. */
+@Qualifier
+annotation class PerformanceMetricsCollection
+
+/** Name of the platform parameter that controls whether to record performance metrics. */
+const val PERFORMANCE_METRICS_COLLECTION = "performance_metrics_collection"
+
+/** Default value for whether to record performance metrics. */
+const val PERFORMANCE_METRICS_COLLECTION_DEFAULT_VALUE = false
+
+/**
+ * Qualifier for the platform parameter that controls the time interval in minutes of uploading
+ * previously recorded performance metrics to the remote service.
+ */
+@Qualifier
+annotation class PerformanceMetricsCollectionUploadTimeIntervalInMinutes
+
+/**
+ * Name of the platform parameter that controls the time interval in minutes of uploading previously
+ * recorded performance metrics to the remote service.
+ */
+const val PERFORMANCE_METRICS_COLLECTION_UPLOAD_TIME_INTERVAL_IN_MINUTES =
+  "performance_metrics_collection_upload_time_interval_in_minutes"
+
+/**
+ * Default value of the time interval in minutes of uploading previously recorded performance
+ * metrics to the remote service.
+ */
+const val PERFORMANCE_METRICS_COLLECTION_UPLOAD_TIME_INTERVAL_IN_MINUTES_DEFAULT_VAL = 15
+
+/**
+ * Qualifier for the platform parameter that controls the time interval in minutes of recording
+ * performance metrics that are to be recorded more frequently.
+ */
+@Qualifier
+annotation class PerformanceMetricsCollectionFrequentRecordTimeIntervalInMinutes
+
+/**
+ * Name of the platform parameter that controls the time interval in minutes of recording
+ * performance metrics that are to be recorded more frequently.
+ */
+const val PERFORMANCE_METRICS_COLLECTION_FREQUENT_RECORD_TIME_INTERVAL_IN_MINUTES =
+  "performance_metrics_collection_frequent_record_time_interval_in_minutes"
+
+/**
+ * Default value of the time interval in minutes of recording performance metrics that are to be
+ * recorded more frequently.
+ */
+const val PERFORMANCE_METRICS_COLLECTION_FREQUENT_RECORD_TIME_INTERVAL_IN_MINUTES_DEFAULT_VAL = 15
+
+/**
+ * Qualifier for the platform parameter that controls the time interval in minutes of recording
+ * performance metrics.
```

Comment:
This should probably clarify how 'regular' differs from 'frequent' since frequent has some additional details above.

Also, what are examples of 'regular' vs 'frequent' again? I'm wondering if we can come up with better terms since these ones aren't terribly self-documenting.

Reply:
Basically, frequent time interval is for those work requests which will occur every 15 mins to collect data related to network, memory and cpu usage. And, regular time interval is for those which will collect data once in a day -- collection of data related to storage. Thoughts @BenHenning?

Reply:
Perhaps instead: low frequency and high frequency metrics?

Reply:
done.